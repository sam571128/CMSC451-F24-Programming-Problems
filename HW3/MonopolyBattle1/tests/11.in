32 1024 1000
19 7
20 29
30 15
29 1
27 31
20 22
28 24
28 32
21 16
29 7
14 31
2 31
24 22
19 10
21 22
22 31
30 14
9 22
28 26
21 19
10 2
9 30
25 9
28 10
25 22
7 24
27 12
4 22
24 9
31 9
25 2
24 18
31 19
11 24
30 3
26 26
31 10
15 2
15 8
10 12
18 3
28 4
6 7
10 26
8 20
32 23
11 31
21 14
8 25
24 24
19 15
11 25
18 29
2 14
17 4
23 31
5 8
1 16
5 2
6 23
26 12
9 3
23 22
7 26
22 18
13 7
18 8
3 1
18 12
15 13
10 4
10 19
6 27
2 30
18 21
19 2
13 5
30 7
23 11
18 4
12 20
13 24
18 26
25 19
5 18
31 2
14 9
32 3
4 28
10 3
28 14
14 12
20 24
23 21
27 28
1 22
26 23
1 10
31 17
18 19
12 16
30 17
28 1
31 18
6 31
27 27
4 31
25 26
14 1
9 25
12 30
23 29
26 5
10 14
5 9
2 19
29 11
7 15
17 2
20 7
25 29
5 5
32 9
22 29
31 8
18 18
19 25
23 10
11 10
5 19
2 13
29 13
7 10
16 11
3 8
19 27
13 32
29 29
25 20
28 29
27 4
9 17
29 16
21 21
14 4
14 29
10 6
14 32
28 17
16 31
27 6
1 5
32 2
15 5
17 25
17 22
12 11
7 19
16 1
30 5
27 8
17 5
10 20
16 6
25 14
26 27
5 31
31 3
7 29
17 9
22 5
28 30
23 5
4 12
5 17
19 17
3 5
26 2
28 31
12 2
17 19
16 17
6 16
10 7
22 27
20 19
29 30
3 9
9 6
31 28
22 12
21 2
11 27
31 12
14 24
14 25
32 31
22 26
7 22
31 20
13 28
25 18
14 7
28 12
4 21
12 22
24 30
16 30
14 26
9 29
12 32
23 25
11 22
30 31
28 25
23 14
28 22
14 10
8 24
29 3
11 8
29 26
14 20
17 32
9 14
30 25
4 13
32 5
4 2
31 13
20 27
6 2
18 32
22 16
12 13
18 5
29 10
30 6
29 18
26 30
7 11
1 17
24 29
24 3
22 21
7 27
5 10
6 5
18 30
18 6
26 13
24 2
24 25
27 9
14 22
32 19
16 9
22 14
2 10
16 25
10 23
30 27
13 2
23 24
1 27
14 16
30 16
11 2
3 28
30 1
11 32
9 16
26 6
29 25
24 11
23 18
19 26
15 23
14 6
19 31
22 23
23 7
16 12
19 32
11 17
15 12
7 12
11 23
19 11
6 1
31 11
23 3
4 29
20 3
28 11
11 19
15 7
9 19
8 30
29 22
5 3
15 28
9 27
16 23
5 25
24 4
3 27
32 18
25 1
25 30
7 16
32 14
24 31
13 17
29 15
30 13
20 32
27 21
10 24
22 2
20 10
17 27
14 3
29 12
23 12
6 8
26 10
9 23
10 29
4 6
20 28
20 16
13 23
26 16
8 31
11 7
2 32
13 12
15 19
6 26
13 19
26 7
17 18
18 11
8 26
15 27
19 12
22 11
8 13
6 4
26 18
8 4
27 1
5 11
9 4
6 22
23 8
26 8
20 25
14 21
9 24
19 18
11 3
8 16
4 14
4 18
5 23
30 2
24 5
16 19
3 32
21 25
16 29
29 32
6 29
31 15
12 1
4 24
6 10
16 16
3 12
19 24
6 15
1 1
11 21
24 12
4 20
23 20
18 27
26 25
31 6
5 16
22 19
1 3
15 25
32 32
3 15
16 21
21 15
22 13
22 1
20 2
17 8
20 15
32 21
30 23
17 12
8 21
9 7
4 9
4 25
10 11
3 17
8 11
3 25
17 24
27 15
18 7
27 26
5 21
10 15
6 9
23 19
31 21
1 4
19 3
23 17
14 15
31 26
28 20
19 16
16 4
7 3
31 29
19 28
17 31
20 5
24 7
26 9
9 10
11 29
20 23
14 8
13 29
2 1
22 6
18 9
12 21
22 3
32 28
2 18
11 16
24 23
18 25
2 20
25 17
5 15
21 8
14 19
27 19
1 15
24 10
25 25
7 2
26 1
19 8
28 28
7 5
17 10
24 32
25 3
3 21
28 23
15 15
2 2
32 8
6 25
30 4
32 22
30 10
7 30
25 8
17 16
29 27
5 6
13 20
15 1
25 16
6 32
32 10
18 17
4 17
16 26
1 30
28 6
9 21
1 20
27 20
4 30
7 7
10 18
7 20
27 5
32 7
24 26
10 25
6 17
21 28
30 9
4 19
12 6
10 31
11 9
29 8
6 21
26 21
15 22
12 10
19 20
19 9
30 18
29 2
21 1
2 28
16 28
31 27
10 28
20 31
30 21
13 14
15 16
2 22
8 17
1 11
23 32
17 29
19 19
13 3
6 24
3 24
9 9
32 6
27 14
10 9
24 1
10 22
11 12
25 24
3 30
9 1
13 10
14 30
17 14
18 24
20 4
9 13
20 13
29 31
26 20
5 1
26 17
12 5
20 8
26 22
14 27
19 6
23 9
3 20
9 8
2 5
1 6
25 5
5 30
15 32
18 1
7 13
23 4
1 9
27 3
1 23
8 3
11 18
11 30
12 12
31 1
4 27
28 19
29 28
29 23
21 4
21 18
30 8
14 23
1 29
12 18
22 8
19 29
12 28
7 1
15 29
1 18
12 14
29 9
9 26
1 7
5 20
13 13
14 14
2 21
8 5
2 8
4 15
10 27
19 30
2 4
12 19
2 9
11 11
1 25
9 12
23 28
6 11
32 16
23 1
12 26
13 21
23 26
17 13
24 16
32 26
4 10
8 8
18 22
7 23
16 5
20 30
12 4
25 23
29 5
28 18
24 21
7 6
28 13
16 18
25 4
19 5
26 19
1 12
17 1
25 28
26 24
24 6
2 23
12 15
23 13
13 9
6 14
22 15
29 24
20 18
6 12
25 32
1 24
16 22
22 20
3 6
16 32
32 30
28 2
21 29
3 23
8 28
8 15
21 7
17 11
25 21
4 7
16 3
15 30
27 23
7 14
30 29
9 28
26 11
3 4
2 11
8 7
16 7
25 13
9 31
23 2
11 4
11 28
15 11
9 2
5 29
6 28
30 32
16 24
5 14
24 17
5 32
14 5
16 8
8 29
25 6
18 10
29 6
13 8
7 9
10 8
26 29
24 28
8 1
14 2
24 20
2 15
23 30
14 17
5 22
4 8
21 12
21 27
23 23
11 26
12 27
10 32
2 26
1 2
13 26
11 14
22 22
18 13
13 4
21 26
6 6
11 20
29 17
29 14
10 10
32 11
6 13
8 14
1 32
7 32
30 22
24 13
11 15
22 7
20 12
32 15
21 30
7 18
21 5
19 23
3 11
23 16
28 16
1 31
2 6
17 20
8 10
17 17
17 7
28 27
12 3
16 15
23 27
4 23
17 21
21 9
23 15
29 4
16 10
4 16
7 31
9 18
16 2
21 24
9 20
32 29
13 22
20 11
18 23
18 28
18 31
16 13
28 7
8 18
13 30
27 25
31 25
27 18
15 26
15 31
26 32
13 1
8 22
7 21
1 26
22 17
8 2
22 25
21 20
21 11
18 15
30 28
10 17
5 26
32 27
27 32
31 7
30 24
21 10
22 4
12 24
22 10
20 9
29 19
19 22
21 6
32 1
20 17
5 27
12 25
6 19
26 3
11 5
28 21
12 31
19 4
20 14
13 25
18 16
7 4
3 10
4 1
27 13
21 23
17 30
21 13
28 9
27 24
27 16
23 6
31 22
7 28
3 16
30 20
2 24
14 13
11 6
27 22
13 18
11 13
14 28
28 15
17 3
2 12
27 29
20 26
15 4
32 12
31 16
15 6
5 28
4 5
22 28
6 20
31 24
7 8
17 23
1 19
15 9
22 30
9 15
10 21
9 5
32 13
27 2
8 19
17 26
30 19
12 9
31 30
19 14
2 16
3 22
31 31
16 14
25 10
31 5
21 31
26 28
2 27
27 17
27 30
26 31
8 6
27 11
31 4
25 7
8 32
10 30
2 7
3 13
28 3
2 25
10 1
31 23
12 7
3 26
25 12
4 11
5 7
15 3
6 3
3 3
19 21
18 20
12 23
2 3
31 14
32 25
1 13
2 29
12 17
26 14
16 20
32 4
3 19
20 20
27 10
13 31
25 11
25 27
5 4
15 21
2 17
19 1
6 30
22 9
4 4
9 32
8 12
15 20
28 5
11 1
29 20
3 18
8 27
3 7
5 24
30 12
13 16
32 24
15 18
4 26
22 32
1 28
1 14
1 21
4 32
17 15
22 24
19 13
13 11
13 6
8 23
24 27
14 11
29 21
26 4
20 1
30 30
25 15
26 15
24 8
3 29
25 31
7 25
28 8
15 10
31 32
30 26
12 29
18 2
10 16
20 21
3 2
24 15
21 32
4 3
7 17
13 15
24 14
20 6
5 12
12 8
3 14
9 11
15 17
10 5
8 9
6 18
18 14
16 27
14 18
3 31
21 3
10 13
17 28
1 8
13 27
15 14
32 20
21 17
30 11
27 7
32 17
17 6
24 19
5 13
15 24
