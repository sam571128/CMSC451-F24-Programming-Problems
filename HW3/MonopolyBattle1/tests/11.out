96 96 96 96 96 147 96 96 96 96 96 96 96 96 96 96 99 96 96 96 96 96 96 96 96 96 136 96 96 96 96 96 